lib/codecs/MultibaseDecoder.js

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import {CborldDecoder} from './CborldDecoder.js';
+import {Base64} from 'js-base64';
+import {encode as encodeBase58} from 'base58-universal';
+
+// this class is used to encode a multibase encoded value in CBOR-LD, which
+// actually means transforming bytes to a multibase-encoded string
+export class MultibaseDecoder extends CborldDecoder {
+  decode({value} = {}) {
+    const {buffer, byteOffset, length} = value;
+    const suffix = new Uint8Array(buffer, byteOffset + 1, length - 1);
+    if(value[0] === 0x7a) {
+      // 0x7a === 'z' (multibase code for base58btc)
+      return `z${encodeBase58(suffix)}`;
+    }
+    if(value[0] === 0x4d) {
+      // 0x4d === 'M' (multibase code for base64pad)
+      return `M${Base64.fromUint8Array(suffix)}`;
+    }
+    return value;
+  }
+
+  static createDecoder({value} = {}) {
+    if(!(value instanceof Uint8Array)) {
+      return false;
+    }
+    // supported multibase encodings:
+    // 0x7a === 'z' (multibase code for base58btc)
+    // 0x4d === 'M' (multibase code for base64pad)
```

Comment:
We only had base64pad so that's what I copied. It's easy to add others -- and if any aren't supported in the compressor then they will just go through as uncompressed strings.